lib/widgets/compose_box.dart

```diff
@@ -372,6 +372,38 @@ class _StreamContentInputState extends State<_StreamContentInput> {
   }
 }
 
+class _StreamTopicInput extends StatelessWidget {
+  const _StreamTopicInput({
```

Comment:
Would `_TopicInput` be a better name? There's nothing in it about letting the user choose a stream.